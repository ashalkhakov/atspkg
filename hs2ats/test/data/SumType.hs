module SumType where

data Num = FloatNum Float
         | IntNum Int
