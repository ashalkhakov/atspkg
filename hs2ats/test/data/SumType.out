datavtype num =
  | Float_num of float
  | Int_num of int
